### PR TITLE
Brushup Agents of Edgewatch Book 1

### PR DESCRIPTION
**Pull request by dogstarrb**
corrected formatting on Almiraj horn
added hardness to Boiling Fountains
Fixed reference to different creature in Kekker and Gref
Corrected Kemeneles AC with Mage Armor spell effect in consideration
Added number of actions for Control Body to Shredskin, added link to Escape in Enshroud ability
Added missing traits on Agonizing Wail for Binumir
added link to Prove Peace activity in Siege Shard's Exorcism ability

### CODE REVIEW

File: packs/data/agents-of-edgewatch-bestiary.db/almiraj.json [hunk 1]
```diff
@@ -202,7 +202,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>Any living creature slain by the almiraj's horn Strike immediately becomes @Compendium[pf2e.conditionitems.Petrified]{Petrified} with no saving throw, in addition to dying.</p>\n<p>A severed almiraj horn retains a wisp of its former magic; in combat, the horn can be wielded as a <em>@Compendium[pf2e.equipment-srd.Dagger]{+1 dagger}</em> or, if affixed to a shaft, a +1 <em>@Compendium[pf2e.equipment-srd.Spear]{spear}</em></p>"
+                    "value": "<p>Any living creature slain by the almiraj's horn Strike immediately becomes @Compendium[pf2e.conditionitems.Petrified]{Petrified} with no saving throw, in addition to dying.</p>\n<p>A severed almiraj horn retains a wisp of its former magic; in combat, the horn can be wielded as a <em>@Compendium[pf2e.equipment-srd.Dagger]{+1 dagger}</em> or, if affixed to a shaft, a <em>@Compendium[pf2e.equipment-srd.Spear]{+1 spear}</em></p>"
```
Comment (TikaelSol) [inline packs/data/agents-of-edgewatch-bestiary.db/almiraj.json:205]:
```suggestion
                    "value": "<p>Any living creature slain by the almiraj's horn Strike immediately becomes @Compendium[pf2e.conditionitems.Petrified]{Petrified} with no saving throw, in addition to dying.</p>\n<p>A severed almiraj horn retains a wisp of its former magic; in combat, the horn can be wielded as a <em>+1 @Compendium[pf2e.equipment-srd.Dagger]{dagger}</em> or, if affixed to a shaft, a <em>+1 @Compendium[pf2e.equipment-srd.Spear]{spear}</em></p>"
```
I think with this one I wouldn't want it to imply that the dragged in item would have the +1.

File: packs/data/agents-of-edgewatch-bestiary.db/siege-shard.json [hunk 2]
```diff
@@ -668,7 +669,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>Mental damage ignores a siege shard's hardness.</p>\n<p>Additionally, proving the siege shard's generative conflict is over with the Prove Peace activity reverts the creature to ordinary stone.</p>"
+                    "value": "<p>Mental damage ignores a siege shard's hardness.</p>\n<p>Additionally, proving the siege shard's generative conflict is over with the @UUID[Compendium.pf2e.actionspf2e.Ma93dpT4K7JbP9gu]{Prove Peace} activity reverts the creature to ordinary stone.</p>"
```
Comment (TikaelSol) [inline packs/data/agents-of-edgewatch-bestiary.db/siege-shard.json:672]: We may need to circle back to this one, I think we've settled on not including one time adventure activities like this one. @SpartanCPA  thoughts?

**Comment by TikaelSol** [comment]
I don't remember how long ago it was but book 1 had a brushup at some point obviously since there's abilities with rule elements on them and those definitely didn't exist when this book was first entered. We may want to replace all spells and equipment on all NPCs as well as part of a brushup though to make sure we have the latest versions of those on them. If you build in dev mode there is a refresh item button that will make it easier to do. Just build the system with `npm run build:dev`, you can stop the build once it's finished with control-c

**Comment by dogstarrb** [comment]
both Almiraj horn and the Siege Shard had things both ways, I guess I just picked the wrong ones to "correct"

**Pull request merged**